lib/ex_doc.ex

```diff
@@ -56,11 +56,12 @@ defmodule ExDoc do
   defp find_formatter(modname) when is_atom(modname), do: modname
 
   defp find_formatter("ExDoc.Formatter." <> _ = name) do
-    Module.concat([name]) |> check_formatter_module(name)
+    [name] |> Module.concat |> check_formatter_module(name)
```

Comment:
I think the rule we have is one |> operator, we put it in one line, and more we break them into several lines.